redfish-client/redfish-client.py

```diff
@@ -314,23 +320,48 @@ def get_manager_info(manager_name, check_SSL):
     logger.info("Arguments parsed")
     logger.debug(arguments)
 
-    # Get $HOME environment.
+    # Get $HOME and $VIRTUAL_ENV environment variables.
     HOME = os.getenv('HOME')
+    VIRTUAL_ENV = os.getenv('VIRTUAL_ENV')
 
     if not HOME:
         print('$HOME environment variable not set, please check your system')
         logger.error('$HOME environment variable not set')
         sys.exit(1)
     logger.debug("Home directory : %s" % HOME)
+    
+    if VIRTUAL_ENV:
+        logger.debug("Virtual env : %s" % VIRTUAL_ENV)
+        
+    # Load master conf file
+    config = ConfigParser.ConfigParser(allow_no_value=True)
+    logger.debug("Read master configuration file")
+    master_conf_file_path = "/etc/redfish-client.conf"
```

Comment:
Hum this one is a bit more tricky as it is really root based.
I'm afraid, using keyword substitution on that one will prevent user for running from the source itself. (I'm doing that really often).

So for this one right now, I would be more in favor of checking for /usr/local/etc/redfish-client.conf or /etc/redfish-client.conf.